These tests only work with read/write access to /dev/tty0 (or similar)

--- a/tests/testsuite.at
+++ b/tests/testsuite.at
@@ -8,7 +8,6 @@
 
 m4_include([unittest.at])
 
-m4_include([utils.at])
 m4_include([libcommon.at])
 m4_include([libkeymap.at])
 m4_include([libkbdfile.at])
